capplets/appearance/appearance-font.c

```diff
@@ -781,6 +801,38 @@ void font_init(AppearanceData* data)
 			  G_CALLBACK (marco_changed),
 			  data);
 
+	/*In a wayland session we must manage MATE font settings for xwayland
+	 *and also manage GNOME font settings for native wayland applications
+	 *so if not running under x11, set the GNOME interface keys too
+	 *Ignore this if for any reason the GNOME schema was not found,
+	 *As we can only set this for compositors using either the MATE or the GNOME gsettings
+	 */
+	if ((!(GDK_IS_X11_DISPLAY ((gdk_display_get_default())))) && (data->interface_gnome_settings))
```

Comment:
Do we need both checks if `data->interface_gnome_settings` is only available under X11 (or when we ant to set it)?

Reply:
Also, there's a lot of parentheses, you could at least drop a pair around `gdk_display_get_default()` :wink:

Reply:
Probably not, was just playing cautious. Again, we load that when NOT user x11 though

Reply:
Anyway, just took that out in the latest force-push

Reply:
Caught those extra  parentheses myself, they were leftovers from the double check I think. Anyway, they are gone, thanks for catching it too.